### PR TITLE
TASK: CGM-5096 create custom Link

### PR DESCRIPTION
**Pull request by FahimNsr**

### CODE REVIEW

**Comment by andrehoffmann30** [comment]
Hi @FahimNsr I miss the compiled plugin code in your PR. You need the commit the compiled plugin code otherwise your changes will not be use, when this package is installed in a project. Please the missing code changes. Thank you.

File: Resources/Private/Translations/de/LinkTypes/Custom.xlf [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+    <file original="" product-name="Sitegeist.Archaeopteryx" source-language="en" datatype="plaintext" target-language="de">
+        <body>
+            <trans-unit id="title" xml:space="preserve" approved="yes">
+                <source>Custom</source>
+                <target>Beliebige Link</target>
```
Comment (andrehoffmann30) [inline Resources/Private/Translations/de/LinkTypes/Custom.xlf]:
```suggestion
                <target>Beliebiger Link</target>
```

File: Resources/Private/Translations/de/LinkTypes/Custom.xlf [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+            </trans-unit>
+            <trans-unit id="custom.label" xml:space="preserve" approved="yes">
+                <source>Custom Link:</source>
+                <target>Beliebige Link:</target>
```
Comment (andrehoffmann30) [inline Resources/Private/Translations/de/LinkTypes/Custom.xlf]:
```suggestion
                <target>Beliebiger Link:</target>
```

File: Resources/Private/Translations/de/LinkTypes/Custom.xlf [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+            </trans-unit>
+            <trans-unit id="custom.placeholder" xml:space="preserve" approved="yes">
+                <source>enter text here</source>
```
Comment (andrehoffmann30) [inline Resources/Private/Translations/de/LinkTypes/Custom.xlf]:
```suggestion
                <source>Enter your text here</source>
```

File: Resources/Private/Translations/de/LinkTypes/Custom.xlf [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+                <target>Hier Text eingeben</target>
```
Comment (andrehoffmann30) [inline Resources/Private/Translations/de/LinkTypes/Custom.xlf]:
```suggestion
                <target>Hier ihren Text eingeben</target>
```

File: Resources/Private/Translations/de/LinkTypes/Custom.xlf [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+            </trans-unit>
+            <trans-unit id="validation.required" xml:space="preserve" approved="yes">
+                <source>this field is required</source>
```
Comment (andrehoffmann30) [inline Resources/Private/Translations/de/LinkTypes/Custom.xlf]:
```suggestion
                <source>This field is required</source>
```

File: Resources/Private/Translations/en/LinkTypes/Custom.xlf [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    <file original="" product-name="Sitegeist.Archaeopteryx" source-language="en" datatype="plaintext">
+        <body>
+            <trans-unit id="title" xml:space="preserve" approved="yes">
+                <source>Custom</source>
+            </trans-unit>
+            <trans-unit id="custom.label" xml:space="preserve" approved="yes">
+                <source>Custom Link:</source>
+            </trans-unit>
+            <trans-unit id="custom.placeholder" xml:space="preserve" approved="yes">
+                <source>enter text here</source>
```
Comment (andrehoffmann30) [inline Resources/Private/Translations/en/LinkTypes/Custom.xlf]:
```suggestion
                <source>Enter your text here</source>
```

File: Resources/Private/Translations/en/LinkTypes/Custom.xlf [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+            </trans-unit>
+            <trans-unit id="validation.required" xml:space="preserve" approved="yes">
+                <source>this field is required</source>
```
Comment (andrehoffmann30) [inline Resources/Private/Translations/en/LinkTypes/Custom.xlf]:
```suggestion
                <source>This field is required</source>
```

**Review by andrehoffmann30 — CHANGES_REQUESTED** [review]
I think the naming is not good here. `Custom` as the only naming information is meaningless. Something like `CustomLink` or `VariableLink` is better. Please think about what type of links your changes allow and choose a meaningful naming.

**Comment by FahimNsr** [comment]
Hi André, I added compiled File, and also as you mentioned, I changed the variable's name to customLink.

**Review by andrehoffmann30 — APPROVED** [review]

**Pull request merged**